### ISSUE TITLE
More help on download.file command

### ISSUE THREAD

**Issue by amadlung** [datacarpentry/R-ecology-lesson#446]
When I first try to download the data set on the very first line of code in http://www.datacarpentry.org/R-ecology-lesson/02-starting-with-data 
RStudio tells me:

Warning messages:
1: In download.file("https://ndownloader.figshare.com/files/2292169",  :
  URL https://ndownloader.figshare.com/files/2292169: cannot open destfile 'data/portal_data_joined.csv', reason 'No such file or directory'

The reason is of course that I haven't made a directory that fits the path provided in the command. I suggest two options for improvement:
1. Add text that tells learner to first make a destination directory. This would assume that the learner knows how to do that and how to edit the code accordingly before downloading the data set.

For example: (Existing: )We are going to use the R function download.file() to download the CSV file that contains the survey data from figshare, and we will use read.csv() to load into memory the content of the CSV file as an object of class data.frame. (New:) Note that you will have to first create the destination directory on your computer so that R will find the place where you want to store the data. You can do that by replacing the placeholder "data/portal_data_joined.csv" in the following command with the full path to an existing directory, such as your Desktop (for example: /Users/username/Desktop/portal_data_joined.csv) - but make sure to replace "username" with your own username. For more information of the path and for making directories see the following link (add a link here to an existing Carpentry lesson that explains paths).

2. Changing the command to: 
download.file("https://ndownloader.figshare.com/files/2292169",
              "/Users/username/Desktop/portal_data_joined.csv")

and the next command to:
surveys <- read.csv ("/Users/username/Desktop/portal_data_joined.csv")
This would require a second set of instructions for Windows users where the path is a bit different.

**Comment by fmichonneau** [comment]
Hello @amadlung 

Thank you for taking the time to send us your suggestions!

We ask learners to set up their working directory and to create a `data` folder inside it in the "Before we start" episode: http://www.datacarpentry.org/R-ecology-lesson/00-before-we-start.html#organizing_your_working_directory

We also generally recommend our learners to use relative paths instead of absolute paths for writing/Reading files to make the code portable.

I'm going to close this, but if you have other suggestions or ideas, please feel free to re-open or open a new issue.
